app/views/user_mailer/password_reset.text.erb

```diff
@@ -0,0 +1,3 @@
+<%= sanitize(t(".content", link: link_to(t".p_reset",
```

Comment:
dùng để chuẩn hóa dữ liệu. Sanitizes HTML input, và lấy hết tất cả tags và attributes mà không phải là whitelisted. Nó đảm bảo an toàn khi sử dụng Unicode, ASCII, và hex characte.